Decisions/scs-xxxx-v1-ssd-flavors.md

```diff
@@ -0,0 +1,252 @@
+---
+title: SSD Flavors
+type: Decision Record
+status: Draft
```

Comment:
To get used to our own standard [SCS-0001](https://github.com/SovereignCloudStack/Docs/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md#document-format), this should be in status `Proposal`. Only the very last commit before merging should
- rename the file
- bump the status to `Draft`

Reply:
You never know which is the last commit before merging, no?
So as soon as the original author thinks that this is ready, can't she assign the status to Draft which becomes then effective as soon as the PR is merged?